sdk/tests/conformance/textures/misc/video-rotation.html

```diff
@@ -89,11 +89,15 @@
                 return;
             }
 
-            const type = 'video/mp4; codecs="avc1.42E01E, mp4a.40.2"';
-            if (!video.canPlayType(type).replace(/no/, '')) {
-                debug(type + " unsupported");
-                return;
-            };
+            let prefer_vp9 = false;
```

Comment:
Please have the test test each video format if it's supported. Thanks.

Reply:
Done.